nautobot/dcim/utils.py

```diff
@@ -65,6 +65,26 @@ def cable_status_color_css(record):
         return CABLE_STATUS_TO_CSS_CLASS.get(status_color, "")
 
 
+def fetch_network_driver_mappings(platform):
+    """
+    Retrieves the network driver mappings for a specific platform.
```

Comment:
Consider making this a method on the `Platform` model instance rather than a standalone function?